lib/shoryuken/polling/base.rb

```diff
@@ -59,8 +59,8 @@ def ==(other)
 
       private
 
-      def delay
-        Shoryuken.options[:delay].to_f
+      def delay(queue)
+        Shoryuken.options[:groups].to_h[queue].to_h.fetch(:delay, Shoryuken.options[:delay]).to_f
```

Comment:
@lucas-ibotta the delay is per group, not per queue, this `Shoryuken.options[:groups].to_h[queue]` is trying to find a delay for a given `queue`. 

Also, for avoiding breaking changes, I would recommend to use `def delay(group = nil)` so if anyone else is using a custom poller and calling `delay` without arguments, it should still work.

Reply:
Yes, nice catch.